uit/gui_tools/submit.py

```diff
@@ -89,28 +108,31 @@ def file_browser_view(self):
     def environment_variables_view(self):
         self.env_names = list()
         self.env_values = list()
+        self.env_browser = list()
+
+        # Might need to create another list
         for i, (k, v) in enumerate(self.environment_variables.items()):
             name_widget = self.env_var_widget(val=k, tag=f'env_key_{i}')
             val_widget = self.env_var_widget(val=str(v), tag=f'env_val_{i}')
+            browser_widget = self.env_file_browser_widget(tag=f'env_browser_{i}')
             self.env_names.append(name_widget)
             self.env_values.append(val_widget)
+            self.env_browser.append(browser_widget)
 
         self.env_names.append(self.env_var_widget(val=None, tag='env_key_-1', placeholder='NEW_ENV_VAR'))
-        self.env_values.append(self.env_var_widget(val=None, tag='env_val_-1', disabled=True))
+        self.env_values.append(self.env_var_widget(val=None, tag='env_val_-1', placeholder=''))
```

Comment:
The reason for this being disabled was so a user couldn't edit the value field without there being a key. Does this make that possible?

Reply:
Unfortunately, adding ```disabled=True``` did not disable the display of that single file browser button for the first row.